themes/theme-b2b/src/components/Switch.styles.ts

```diff
@@ -4,9 +4,9 @@ export const Switch: ThemeComponent<'Switch'> = {
   container: cva(''),
   track: cva(
     [
-      'bg-bg-surface-underlay shadow-shadow-light shadow-[0_0_0_1px]',
```

Comment:
<img width="1158" alt="Captura de pantalla 2024-01-16 a la(s) 20 05 01" src="https://github.com/marigold-ui/marigold/assets/156671137/607c1163-4235-43d3-a9e3-bb155941b1a3">

This piece of code has seemingly been correctly updated, however, there is still a manual shadow used later in the same line of code. Is that intentional?

Reply:
I have removed the shadow